packages/react-native-editor/src/test/index.test.js

```diff
@@ -0,0 +1,185 @@
+/**
+ * External dependencies
+ */
+import { AppRegistry } from 'react-native';
+// eslint-disable-next-line import/no-unresolved
+import { render, waitFor } from 'test/helpers';
```

Comment:
> I had to disable `import/no-unresolved` rule for this import statement because it's not recognized by the import resolver ([reference](https://github.com/WordPress/gutenberg/blob/trunk/tools/eslint/import-resolver.js)).

Why is [the ESLint issue](https://github.com/WordPress/gutenberg/pull/37073#discussion_r775835418) just now a problem? Haven't we used `import * from 'test/helpers';` plenty of times before? Is there something different about this particular file or context? 

I wonder if there is an alternative to disabling the ESLint rule. Could we define the alias in a location ESLint utilizes? Could we leverage a different name/location for the `test/helpers` file, e.g. `@wordpress/react-native-editor`?

Reply:
> Why is the ESLint issue just now a problem? Haven't we used import * from 'test/helpers'; plenty of times before? Is there something different about this particular file or context?

Right, I forgot to mention that only test files without `.native.js` extension are being affected by this ESLint issue. Files with extension `.native.js`, `.android.js`, and `.ios.js` would produce the same error but this is not the case because that rule is disabled for them at ESLint configuration level:

https://github.com/WordPress/gutenberg/blob/ca85ced9298a252d4832438d8408ea4cda95696f/.eslintrc.js#L171-L185

In fact, the file `packages/react-native-editor/src/test/index.test.js` should be included in that rule override, but for an unknown reason, it's not working for files under `react-native-editor` path. Maybe it's related to the fact that there's already an ESLint configuration file under that package ([reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/react-native-editor/.eslintrc.js)). Being this said, I'm thinking to apply the same overriding in that file to match the same configuration 🤔 .

> I wonder if there is an alternative to disabling the ESLint rule. Could we define the alias in a location ESLint utilizes? Could we leverage a different name/location for the test/helpers file, e.g. @wordpress/react-native-editor?

We could update the import resolver ([reference](https://github.com/WordPress/gutenberg/blob/trunk/tools/eslint/import-resolver.js)) and omit imports pointing to `test/helpers` path, but I'm not very convinced applying this modification just for a module mapped to be used only in tests.

Additionally, in Gutenberg looks like the development files, including the test files, are considered to be less strict in terms of linting, following the same rule disabling as for JS native files:

https://github.com/WordPress/gutenberg/blob/ca85ced9298a252d4832438d8408ea4cda95696f/.eslintrc.js#L22-L32

https://github.com/WordPress/gutenberg/blob/ca85ced9298a252d4832438d8408ea4cda95696f/.eslintrc.js#L171-L185

Therefore, matching the behavior of the web version, I think it should be safe just to disable that rule for test files.

Reply:
> Being this said, I'm thinking to apply the same overriding in that file to match the same configuration 🤔 .

@dcalhoun I finally decided to instead of disabling the rule here, I override the rule in the `react-native-editor` ESLint configuration in https://github.com/WordPress/gutenberg/pull/37073/commits/9393b7a7f425fc812bcb774ea7740059b85951c7.